workflows/diagnostics/fv3net/diagnostics/offline/_helpers.py

```diff
@@ -143,11 +150,32 @@ def insert_rmse(ds: xr.Dataset):
 
 
 def load_grid_info(res: str = "c48"):
+    if gsrm_name_from_resolution_string(res) == "scream":
+        return load_grid_info_scream(res)
+    elif gsrm_name_from_resolution_string(res) == "fv3":
+        return load_grid_info_fv3(res)
+    else:
+        raise ValueError(f"Unknown evaluation grid {res}.")
+
+
+def load_grid_info_fv3(res):
     grid = catalog[f"grid/{res}"].read()
     wind_rotation = catalog[f"wind_rotation/{res}"].read()
     land_sea_mask = catalog[f"landseamask/{res}"].read()
     grid_info = xr.merge([grid, wind_rotation, land_sea_mask])
-    return safe.get_variables(grid_info, GRID_INFO_VARS).drop_vars(
+    return safe.get_variables(grid_info, GRID_INFO_VARS_FV3).drop_vars(
+        "tile", errors="ignore"
+    )
+
+
+def load_grid_info_scream(res):
+    grid = catalog[f"grid/{res}"].read()
+    land_sea_mask = catalog[f"landseamask/{res}"].read()
+    grid_info = xr.merge([grid, land_sea_mask])
+    # TODO: currently uses the land model's grid,
+    # but should use scream's when we have it
+    grid_info = grid_info.rename({"lndgrid": "ncol"})
```

Comment:
should we just rename in the saved zarr?

Reply:
Yes, I don't want to mix it here because the file needs to be updated to the atmosphere's mask instead of using land model's. Will open a separate PR as soon as I get that file.